CMakeLists.txt

```diff
@@ -131,6 +127,11 @@ if(reporting)
   )
 endif()
 
+if (coverage)
+  include (CodeCoverage)
+  include (RippledCov)
+endif()
```

Comment:
No spaces between function name and parentheses, please. This style is not yet consistent throughout our CMake, but it is the mandatory style for all future changes (matching CMake's documentation). Please fix the other changes in this PR too.

Reply:
Will fix that, thanks